src/main/scala/scala/collection/decorators/IterableDecorator.scala

```diff
@@ -33,4 +33,46 @@ class IterableDecorator[C, I <: IsIterable[C]](coll: C)(implicit val it: I) {
   def lazyFoldRight[B](z: B)(op: it.A => Either[B, B => B]): B =
     it(coll).iterator.lazyFoldRight(z)(op)
 
+
+  /**
+    * Constructs an iterator where consecutive elements are accumulated as
```

Comment:
We should replace “iterator” with “collection”

Reply:
done :)